src/main/java/com/social/instagram/config/RedisFeedsCacheManagerConfig.java

```diff
@@ -17,11 +17,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import static com.social.instagram.util.cache.RedisConstants.USER_ID_FEEDS_CACHE_EXPIRE_MINUTE;
-import static com.social.instagram.util.cache.RedisConstants.USER_ID_FEEDS_CACHE;
+import static com.social.instagram.util.cache.RedisFeedsConstants.USER_ID_FEEDS_CACHE_EXPIRE_MINUTE;
+import static com.social.instagram.util.cache.RedisFeedsConstants.USER_ID_FEEDS_CACHE;
 
 @Configuration
-public class RedisCacheManagerConfig {
+public class RedisFeedsCacheManagerConfig {
```

Comment:
`RedisCacheManager`는 고유한 이름이므로 피드를 앞에 붙여주는게 좋을 것 같아요~

```suggestion
public class FeedsRedisCacheManagerConfig {
```